CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Added
 
+- Added an ability to modify enabled vcpu features for aarch64 uVMs
```

Comment:
s/vcpu/vCPU/, s/uVMs/microVMs/ .
I think we can provide examples of what vCPU features can be enabled and a little bit of context on why users may want to modify checked KVM caps.

Reply:
I don't think adding documentation/examples to the `CHANGELOG` is correct idea.

Reply:
why it is not a good idea? Can we point at least to See <doc> to point a reader to which feature a customer can add?
In addition to that i would suggest to:

Added the ability for an user to specify a set of vCPU features <where/when?> to be enabled/disabled when booting a microVM on aarch64 system. For the list of available feature please see <doc where is explained in more details the feature available>.

Reply:
I still don't think this should be in `CHANGELOG`. It makes sense to add it to the `release notes` or a release blog post or smth similar.
`CHANGELOG` is just a list of bullet points with only essential info about changes.
Updated the `CHANGELOG` entry.